stream_alert_cli/test.py

```diff
@@ -607,6 +607,7 @@ def __init__(self, config, context):
         self.kms_alias = 'alias/stream_alert_secrets_test'
         self.secrets_bucket = 'test.streamalert.secrets'
         self.outputs_config = load_outputs_config()
+        self.config = config
```

Comment:
suggestion: don't cache the whole config and instead just get the region from it here once (also avoids having to do the lookup a bunch throughout code below).